app/frontend/styles/provider/_recruitment_performance_report.scss

```diff
@@ -1,11 +1,16 @@
 .recruitment-performance-report-table {
+  &__wrapper {
+    padding-bottom: govuk-spacing(4);
+    overflow-x: scroll;
```

Comment:
Good catch @inulty-dfe! @elceebee this screenshot also shows that we'd want to use a margin, rather than padding. With margin, I'd expect the scrollbar would be at the bottom of the table 👌